ecommerce-demo/workspaces/childDemo_bigboxretailer/workspaceDataFilterSettings.json

```diff
@@ -0,0 +1,46 @@
+{
+  "workspaceDataFilterSettings": [
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-bigboxretailer",
+        "attributes": {
+          "title": "ecommerce merchant bigboxretailer",
+          "filterValues": [
+            "merchant__bigboxretailer"
+          ]
+        },
+        "relationships": {
+          "workspaceDataFilter": {
+            "data": {
+              "id": "WDF__CLIENT_ID",
+              "type": "workspaceDataFilter"
+            }
+          }
+        }
+      }
+    },
+    {
+      "data": {
+        "type": "workspaceDataFilterSetting",
+        "id": "ecommerce-merchant-bigboxretailer-cat",
```

Comment:
Should we add `demo_wdfSettings` as a prefix for the id? Also should we change hyphens `-` to underscores `_`, to be consistent with the naming of other ids (e.g., the colorPaletteId and themeId)?

Something like:

```json
"id": "demo_wdfSettings_ecommerce_merchant_bigboxretailer_cat",
```